_codux/boards/toast/toaster.board.tsx

```diff
@@ -0,0 +1,47 @@
+import { createBoard } from '@wixc3/react-board';
+import toast, { Toaster } from 'react-hot-toast';
+import { Toast } from '~/src/components/toaster/toast/toast';
+
+import styles from './toaster.board.module.scss';
+
+export default createBoard({
+    name: 'Toaster',
+    Board: () => {
+        const handleTopToast = () => {
+            toast('Toast notification at the top of the page', {
+                position: 'top-center',
```

Comment:
I can add here more useful options and comment them as hints for the user what they can do. Does it make sense?